scripts/start-deployCanyon

```diff
@@ -9,20 +9,19 @@ isDebugging && set -x
 : "${CANYON_BUILD:=lastSuccessfulBuild}"
 
 if [ "${VERSION}" != "b1.7.3" ]; then
-  log "ERROR: Canyon server type only supports VERSION=b1.7.3"
+  logError "Canyon server type only supports VERSION=b1.7.3"
   exit 1
 fi
 
-canyonJob="https://jenkins.glass-launcher.net/job/Canyon"
+canyonJob="https://canyonmodded.p0rtal.gay/job/Canyonmodded"
```

Comment:
Revert URL change

Reply:
it does not work anymore with this url but ok